[2025-03-16T03:53:00.408+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-16T03:53:00.423+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.bronze_layer_group.ingest_raw_data manual__2025-03-16T03:48:17.725362+00:00 [queued]>
[2025-03-16T03:53:00.432+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.bronze_layer_group.ingest_raw_data manual__2025-03-16T03:48:17.725362+00:00 [queued]>
[2025-03-16T03:53:00.432+0000] {taskinstance.py:2866} INFO - Starting attempt 4 of 7
[2025-03-16T03:53:00.442+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): bronze_layer_group.ingest_raw_data> on 2025-03-16 03:48:17.725362+00:00
[2025-03-16T03:53:00.450+0000] {standard_task_runner.py:72} INFO - Started process 630 to run task
[2025-03-16T03:53:00.453+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'bronze_layer_group.ingest_raw_data', 'manual__2025-03-16T03:48:17.725362+00:00', '--job-id', '123', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmpde9i72yg']
[2025-03-16T03:53:00.454+0000] {standard_task_runner.py:105} INFO - Job 123: Subtask bronze_layer_group.ingest_raw_data
[2025-03-16T03:53:00.498+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline.bronze_layer_group.ingest_raw_data manual__2025-03-16T03:48:17.725362+00:00 [running]> on host c891f3a5df31
[2025-03-16T03:53:00.580+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='bronze_layer_group.ingest_raw_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-16T03:48:17.725362+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-16T03:48:17.725362+00:00'
[2025-03-16T03:53:00.580+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-16T03:53:00.583+0000] {ingest_raw_data.py:139} INFO - Scanning for files in topics/tracking.user_behavior.validated/year=2025/month=01...
[2025-03-16T03:53:00.584+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2025-03-16T03:53:00.586+0000] {base.py:84} INFO - Retrieving connection 'minio_conn'
[2025-03-16T03:53:00.587+0000] {warnings.py:109} WARNING - <string>:9: UserWarning: AWS Connection (conn_id='minio_conn', conn_type='s3') expected connection type 'aws', got 's3'. This connection might not work correctly. Please use Amazon Web Services Connection type.

[2025-03-16T03:53:00.587+0000] {connection_wrapper.py:325} INFO - AWS Connection (conn_id='minio_conn', conn_type='s3') credentials retrieved from login and password.
[2025-03-16T03:53:01.087+0000] {ingest_raw_data.py:150} INFO - Loading checkpoint data...
[2025-03-16T03:52:59.350+0000] {ingest_raw_data.py:154} INFO - No new files to process
[2025-03-16T03:52:59.351+0000] {python.py:240} INFO - Done. Returned value was: {'data': [], 'skipped_files': 1}
[2025-03-16T03:52:59.382+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-16T03:52:59.383+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=bronze_layer_group.ingest_raw_data, run_id=manual__2025-03-16T03:48:17.725362+00:00, execution_date=20250316T034817, start_date=20250316T035300, end_date=20250316T035259
[2025-03-16T03:52:59.454+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-16T03:52:59.494+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-16T03:52:59.496+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
